tutorials/cp-portal-cloud-foundry-getting-started/cp-portal-cloud-foundry-getting-started.md

```diff
@@ -12,19 +12,18 @@ author_profile: https://github.com/LindsayBert
 <!-- description --> To get started with building a launchpad site in the SAP Launchpad service, you must perform the required onboarding steps.
 
 ## Prerequisites
-  - If you want to use a trial environment, you need to first register it. You can register to a trial account using this link: [Create a trial account](https://account.hanatrial.ondemand.com/trial/#/home/trial). For a more detailed explanation, see this tutorial: [Get a Free Account on SAP BTP Trial](hcp-create-trial-account)
+  - If you want to use a trial environment, you need to first register it. You can register to a trial account using this link: [Create a trial account](https://www.sap.com/israel/products/business-technology-platform/trial.html). Scroll down and click **Get trial now**.
   - If you're using a production environment, you should have a subaccount configured. If you don't have a configured subaccount, refer to this topic: [Initial Setup](https://help.sap.com/viewer/8c8e1958338140699bd4811b37b82ece/Cloud/en-US/fd79b232967545569d1ae4d8f691016b.html).
 
 
-## Intro
-Note that you can also do this tutorial using a SAP BTP free plan option. To do this, please follow this tutorial: [Set Up SAP Launchpad Service Using the Free Tier Model for SAP BTP](cp-portal-cloud-foundry-setup-freetier).
-
-
 ## You will learn
   - How to subscribe to the SAP Launchpad service
   - How to assign yourself to the `Launchpad_Admin` role so that you can create and manage sites in the SAP Launchpad service
   - How to access the SAP Launchpad service
 
+## Intro
+  Note that you can also do this tutorial using a SAP BTP free plan option. To do this, please follow this tutorial: [Set Up SAP Launchpad Service Using the Free Tier Model for SAP BTP](cp-portal-cloud-foundry-setup-freetier).
```

Comment:
I think that you should move this line to the note below and use a proper label for the note, for example, "Infor", "Tip", etc.